### DISCUSSION TITLE
Why display premium volume for options instead of notional volume?

### DISCUSSION THREAD

**Discussion by 0xngmi** [DefiLlama/DefiLlama-Adapters#12617] [General]
DefiLlama used to display notional volume for options but we had to change it.

The issue with notional is that it's extremely easy to fake, so it does get faked and then the data becomes almost useless.

For example, aevo rewarded volume so there were a few users that were trading options that were extremely OTM, like 100M notional with a  premium of 500$, so only 500$ were getting traded but 100M showed up as notional volume.

This led to insanely inflated volumes, which resulted in billions of options volume, which were all fake. This made the metrics completely useless and by looking at the charts it was impossible to tell which direction was it growing in or which protocols were leading.

Because of that we switched to preferring premiums over notional, we still provide notional data and users that want it can chart it, but the default on our site is to use premiums.

**Comment by ugolino** [comment]
I'd say Notional makes more sense because:

- most of CEXs/tradfi use notional, so easier to compare DeFi vs CEXs
- Comparing options vs. perps is difficult if one uses notional and the other uses premium. If we use premium for options, we should use funding for perps, for example.

Using premium gives the entire options sector a weaker appearance, which does not seem fair.

I agree that showing notional can be gamed, making comparisons among options protocols less reliable, but it still provides a fairer benchmark against other products and CEXs.

**Reply by 0xBobdbldr** [reply]
100%, and it's more downside to us the projects rather than upside to data cleaning, as "on paper" the notional volume and fees paid are correct regardless of OTMoneyiness of the strike itself

**Comment by 0xMughal** [comment]
Agree with Dan, metrics can be gamed with any sort of trading, spot, perps, options. It's quite easy to spot this and DeFillama has done a good job on making it easy to spot. 

I think we need notional volume to help give users a better idea of how options are doing overall. 

Wonder if there is a way we can flag protocols 'gamifying' the numbers for extended periods of time?
